addons/website/views/snippets/s_website_form.xml

```diff
@@ -2,9 +2,15 @@
 <odoo>
 
 <template id="s_website_form" name="Form">
-    <section class="s_website_form pt16 pb16" data-vcss="001" data-snippet="s_website_form">
-        <div class="container">
+    <section class="s_website_form o_cc o_cc2 pt64 pb64" data-vcss="001" data-snippet="s_website_form">
+        <div class="o_container_small">
             <form action="/website/form/" method="post" enctype="multipart/form-data" class="o_mark_required" data-mark="*" data-pre-fill="true" data-model_name="mail.mail" data-success-mode="redirect" data-success-page="/contactus-thank-you">
+                <div class="row text-center">
+                    <div class="col pb16">
```

Comment:
The editor has some sort of issue when using `col` classes "without size".
I don't get any problem (and neither recall what it was this issue all about honestly), but let's just be sure and use a `col-12` here, thanks!